runtime/bindings/python/tests/test_inference_engine/test_function.py

```diff
@@ -2,13 +2,15 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import numpy as np
+from numpy.core.defchararray import array
 import pytest
 
 import openvino.opset8 as ops
 
-from openvino import Function
-from openvino.descriptor import Tensor
-from openvino.impl import PartialShape
+from openvino import Function, Tensor
+from openvino.descriptor import Tensor as TensorDescriptor
```

Comment:
Could you please import it like ```DescriptorTensor```? We have ```TensorDescriptor``` in the ols API and it may be a little bit confusing.
CC @jiwaszki @ilyachur

Reply:
Changed 😊